incubator/elasticsearch/values.yaml

```diff
@@ -35,8 +35,11 @@ master:
   name: master
   replicas: 3
   heapSize: "512m"
-  storage: "4Gi"
-  # storageClass: "ssd"
+  persistence:
+    enabled: true
+    accessMode: ReadWriteOnce
+    storage: "4Gi"
```

Comment:
Please rename to `size`, only for consistency w/ other charts.

Reply:
WIll update to size.